src/Symfony/Component/DependencyInjection/Compiler/AnalyzeServiceReferencesPass.php

```diff
@@ -140,11 +140,21 @@ protected function processValue($value, $isRoot = false)
         $this->byConstructor = true;
         $this->processValue($value->getFactory());
         $this->processValue($value->getArguments());
+
+        // "wither" calls are part of the constructor-instantiation graph
```

Comment:
`"with-like mutators"`?

Reply:
That's not a mutator, and there term "wither" already exists for that concept elsewhere, so I'm good with the vocab personnaly.

Reply:
factory*. It's just that wither means withering (for a plant for example). The term would certainly work with any French person, as the construction is very "English like", but it's gonna confuse for sure English native speakers

Reply:
Some use it for what we mean here :)
https://projectlombok.org/features/experimental/Wither